docs/use-cases/exchange.md

```diff
@@ -0,0 +1,138 @@
+
+### 2.1 Integration Scope
+
+The scope for integrating with the OMG Network by a given exchange includes the following steps:
+
+#### 2.1.1 Generate ETH Addresses
+
+As an exchange, you need to be able to generate new Ethereum addresses for your customers. The OMG Network reflects the same addresses as the Ethereum network, thus you don't need to implement extra logic for your platform's wallet. If you already have Ethereum supported, you shouldn't have any issues.
+
+Otherwise, you can create a new public/private key pair using [`web3.eth.accounts.create`](https://web3js.readthedocs.io/en/v1.2.0/web3-eth-accounts.html#create) function provided by the web3 library. 
+
+> This document doesn't cover best practices for managing and storing keys due to a variety of methods different exchanges use to achieve a high level of security guarantees on their platform.
+
+#### 2.1.2 Query Blockchain
+
+It's important to understand how to query data from the OMG Network. There are several things that might be important for any exchange:
+- Transaction details - use `getTransaction(id)` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+- Block details - use `getTransactions(filters)` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+- Address balance - use `getBalance(address)` function by the [`omg-js`](https://docs.omg.network/omg-js/). For more details, check [`Retrieve Balances`](https://docs.omg.network/network/balances) guide
+- Network fees - use `getFees` function by the [`omg-js`](https://docs.omg.network/omg-js/)
+
+You can find more details about these functions using the following resources:
+- [omg-js library](https://github.com/omgnetwork/omg-js) - the official JavaScript library that allows to implement the required functionality
+- [omg-js documentation](https://docs.omg.network/omg-js/) - a list of all functions of the omg-js library
+- [omg-js tutorials](https://docs.omg.network/network/start) - installation process, code samples, and various tips for using the omg-js library
+- [omg-js-samples](https://github.com/omgnetwork/omg-js-samples) - a JavaScript project that contains implementation for the core features of the OMG Network
+- [blockchain explorer](https://docs.omg.network/environments) - a tool that allows to view transactions and blocks that happen on the network
+
+#### 2.1.3 Make Deposits
+
+The `Deposit` terminology for an exchange has a different meaning from a deposit on the OMG Network. It's very important to distinguish between those two:
+1. OMG Network deposit - represents an ETH transaction to the corresponding `Vault` smart contract on the Ethereum network. This creates a new deposit on the OMG Network and allows the use of these funds until a user decides to withdraw them back to Ethereum.
```

Comment:
change `allows the use of these funds` to `allow the depositor to transact on the network`?